enterprise/models.py

```diff
@@ -4344,6 +4387,37 @@ def membership_user(self):
         """
         return self.enterprise_customer_user or self.pending_enterprise_customer_user
 
+    @property
+    def member_email(self):
+        """
+        Return the email associated with the member
+        """
+        if self.enterprise_customer_user:
+            return self.enterprise_customer_user.user_email
+        return self.pending_enterprise_customer_user.user_email
+
+    @property
+    def member_status(self):
+        """
+        Return the status of the membership
+        """
+        if self.is_removed:
+            return "removed"
+        if self.enterprise_customer_user:
+            return "accepted"
+        return "pending"
```

Comment:
These should be constants.  Also, this is a good indication that maybe you want to add a field called `status` to your membership model.